src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -58,7 +58,14 @@ fieldset {
   }
 }
 
-.va-nav-breadcrumbs-list, legend, .nav-header > h4, .nav-header, .usa-input-error, .input-error-date, .usa-alert {
+.va-nav-breadcrumbs-list,
+legend,
+.nav-header > h4,
+.nav-header > h2,
```

Comment:
Here is the new selector that was added.

Reply:
This looks very low risk, since it's wrapped in the `.nav-header` class. We used to have an H4 where the H2 is now, and have experimented with focus management in the past to improve handling with NVDA.

As I was testing the PR 14024, it occurred to me why I'm such a proponent of removing the focus halo on these headings with tabIndex -1. The screen readers usually have their own highlight (light green background for JAWS, black outline for VO, NVDA has a plugin that a lot of folks use to add a red outline), so we'd be doubling up. I stand by my original call to remove the halo in this case, and will revise if the design system council feels differently.

Reply:
Gotcha. Also, in the letters app they have some custom code going on that still uses an `<h4>` inside a `.nav-header` div, so I left that older style in.